### PR TITLE
Added a cluster-only option

### PR DESCRIPTION
**Pull request by malscent**

### CODE REVIEW

File: common/constants.sh [hunk 1]
```diff
@@ -17,6 +17,7 @@ export STARTUP=0
 export SYNC_GATEWAY=0
 export WAIT=0
 export NO_CLUSTER=0
+export CLUSTER_ONLY=0
```
Comment (patrick-stephens) [inline common/constants.sh:20]: Probably could do with documenting these as not overly clear with, no, do and only

File: readme.md [hunk 2]
```diff
@@ -136,10 +136,16 @@ This project uses [BATS](https://bats-core.readthedocs.io/en/latest/) for unit t
...
-**Usage**: ```./main.sh -```
+**Usage**: ```./main.sh -h```
 
 **Purpose**:  Prints help regarding the parameters to pass to this script
 
+### ```-co|--cluster-only```
+
+**Usage**: ```./main.sh --cluster-only```
+
+**Purpose**:  Skips install for pre-installed images
```
Comment (patrick-stephens) [inline readme.md:147]: I think this needs expanding a bit, what does it mean and what actually happens? Why would you ever install if pre installed?

**Review by patrick-stephens — APPROVED** [review]

File: common/constants.sh (hunk 1 above)
Comment (malscent) [reply common/constants.sh:22]: Yep.. added

File: readme.md (hunk 2 above)
Comment (malscent) [reply readme.md]: Yep.. expanded on it a bit

**Pull request merged**